challenges/advanced-unnamed-args/question.py

```diff
@@ -0,0 +1,41 @@
+"""
+TODO:
+    Define a callable type that accepts a string parameter and returns None.
+    *The parameter name can be any.*
+
+HINT: Use Protocol
+"""
```

Comment:
It's bit difficult to understand what I need to do reading the instruction. Maybe we can define the callable but leave its type empty, or any other way that makes it easier to start.

Reply:
The argument name matters(it must start with double underscore) in this challenge. BTW here is the solution, any suggestion in making the requirement clear?

```python
class SingleStringInput(Protocol):
    def __call__(self, __arg: str) -> None:
        ...
```

Reply:
I realized this question can be solved by `Callable`, so I adjusted the difficulty for this.

I will think of another challenge to test the unnamed argument.